protocol/routing/README.md

```diff
@@ -0,0 +1,74 @@
+IPFS Routing Protocol Spec
+==========================
+
+Authors: David Dias
+
+Reviewers:
+
+TODOS:
+
+-----------------------
+
+> This spec defines the routing protocol spec. Routing offers an interface for the features exposed by `Peer discovery` and `DHT`. The spec is a **Work In Progress**.
+
+## Supports
+
+- Routing primitives
+  - Publish and fetch content (also providing)
+- Maintaining partial state of the network
+  - DHT
+  - kbucket
+
+## Overview
+
+The Routing Protocol is composed by three componenets, these are:
+- Interface: Our routing primitives that are offered for the user, such as finding and publishing content, including the storage and lookup of metadata (Providers).
+- Peer Discovery: Responsible for filling our kbucket with best candidates.
```

Comment:
Apologies, this `/routing` folder is from very long ago, before we moved Peer Routing to part of the network layer, I forgot to explicit deprecated it as there is lot of incoherences, I'll just delete it all together. The holy grail for all things network is on `/network`